x-pack/legacy/plugins/code/public/components/main/main.scss

```diff
@@ -293,3 +293,26 @@
   border-left: $euiBorderThin;
   width: 300px !important;
 }
+
+.codeBranchSelectorTabs {
+  max-height: calc(100vh - 19rem);
+  overflow: auto;
+}
+
+.codeBranchSelectorTabContentContainer {
+  padding-top: $euiSizeS;
+}
+
+.codeBranchSelectorPopover {
+  margin-top: $euiSizeXS;
+  padding-top: $euiSizeS !important;
```

Comment:
_If possible_, try to increase the specificity of this selector instead of using `!important`

Same below 👇

Reply:
have to using `!important` to override default `eui` styles